lib/edition_filter.rb

```diff
@@ -22,6 +22,7 @@ def editions
                    .preload(revision_joins, :status, :document, :last_edited_by)
     scope = access_limited_scope(scope)
     scope = filtered_scope(scope)
+    scope = remove_pre_release_document_type_scope(scope)
```

Comment:
Minor: these methods seem to be phrased in "noun-form" - `remove` is a verb - so `non_pre_release_types_scope` would be more consistent.

Reply:
cool, will do!